src/bindings/pyrpr/src/pyrpr.py

```diff
@@ -95,67 +91,34 @@ def wrapped(*argv):
 
 
 class _init_data:
-    _log_fun = None
+    log_fun = None
+    lib_wrapped_log_calls = False
 
 
-def init(log_fun, rprsdk_bin_path=None):
+def init(lib_dir, log_fun, lib_wrapped_log_calls):
+    _init_data.log_fun = log_fun
+    _init_data.lib_wrapped_log_calls = lib_wrapped_log_calls
 
-    _module = __import__(__name__)
+    lib_name = {
+        'Windows': "RadeonProRender64.dll",
+        'Linux': "libRadeonProRender64.so",
+        'Darwin': "libRadeonProRender64.dylib"
+    }[platform.system()]
 
-    _init_data._log_fun = log_fun
```

Comment:
Why is this change needed in a RIF change?

Reply:
I did that change as a code cleanup several months ago and just cherry picked commits.